cmds/core/chroot/chroot_test.go

```diff
@@ -159,7 +158,10 @@ func TestGroupsString(t *testing.T) {
 }
 
 func TestChroot(t *testing.T) {
-	testutil.SkipIfNotRoot(t)
```

Comment:
@rminnich do you have context on the deprecation of testutil?  I understand that we want to get rid of some stuff like `testutil.Run`, etc, where possible - but to me it seems like `SkipIfNotRoot` and `SkipIfInVMTest` are useful to helpl a bunch of code duplication in tests?   Do you have thoughts on this?

Reply:
I'll be honest -- I don't know who or what is driving that deprecation effort. Anyone?

Reply:
Personally, I do not see a huge difference between two lines of code being duplicated in multiple tests vs an invocation of a wrapper function implementing these two lines. However, this is just personal preference. I will drop the commit for now and move in the other changes.